_glossary/workload-name.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Workload
+type: markdown
+---
+A unique name for a **Workload**, identifying it within the **Service Mesh**.
+Unlike **Service Name** and **Workload Principal**, **Workload Name** is not a strongly verified property and should not be used when enforcing ACLs.
+  * **Workload Names** are accessible in Istio configuration as the `source.name` and `destination.name` attributes.
```

Comment:
Shame this isn't source.workload.name and destination.workload.name.

Reply:
If it matters, we've been using `source.uid` and `destination.uid` to denote the workload IDs.

Reply:
So what's the relationship between source.name and source.uid? Should we have a glossary entry for Workload Id? Given workload id, why do we need workload name?

Reply:
Added a glossary entry for Workload ID.

Both are useful, since workload name is a natural grouping of workload IDs and more useful in dashboards.